testdata/BUILD

```diff
@@ -276,6 +277,12 @@ container_flatten(
     image = ":link_with_files_base",
 )
 
+# container_test(
+#     name = "structure_test",
+#     configs = ["test.yaml"],
+#     image = ":link_with_files_base",
+# )
```

Comment:
Does the `tar` driver need Docker?

I'd actually like to see us exercise a few behaviors of the test framework here (these kind of serve as samples I often link to as well).

Can you create a `tar` driver that should run anywhere, and follow @damienmg guidance for running with Docker on Bazel CI?

Reply:
Nope, the point of the tar driver is that it explicitly doesn't use Docker (which is why command tests don't currently work with it). I'll add a run of some file tests using the tar driver, and then try and use @damienmg's suggestion to get another run working with Docker.

Reply:
Can we uncomment these if we make them use the `tar` driver?